include/aca_ovs_control.h

```diff
@@ -221,6 +224,8 @@ class ACA_OVS_Control {
   private:
   ACA_OVS_Control(){};
   ~ACA_OVS_Control(){};
+
+  uint32_t oam_server_port = 8300;
```

Comment:
Please don't initialize it to a hard coded value, give compile a change to catch error if it is used without someone setting it.